policies/attestation_type.rego

```diff
@@ -2,22 +2,22 @@ package policies.attestation_type
 
 import data.lib
 
+# Currently this is the only type we know about
+known_types := ["https://in-toto.io/Statement/v0.1"]
+
+# METADATA
+# custom:
+#   short_name: unknown_att_type
+#   name: An unknown attestation type was found
```

Comment:
Perhaps it would make sense to take advantage of the [builtin annotations](https://www.openpolicyagent.org/docs/latest/annotations/#annotations) (`title` and `description`)